docker-compose.yml

```diff
@@ -112,7 +112,7 @@ services:
     build:
       dockerfile: packages/hoppscotch-backend/Dockerfile
       context: .
-      target: prod
+      target: dev
```

Comment:
Is this change intentional ?

Reply:
Yup, it is done during dev. After the PR Is reviewed I revert it back to prod before merge.